azext_iot/tests/iothub/__init__.py

```diff
@@ -48,6 +58,21 @@
 MAX_RBAC_ASSIGNMENT_TRIES = settings.env.azext_iot_rbac_max_tries or 10
 ROLE_ASSIGNMENT_REFRESH_TIME = 120
 
+# Endpoints
+EP_RG = settings.env.azext_iot_ep_rg or ENTITY_RG
```

Comment:
we should think of a neater way to create many constant variables. Prob a future task.

Reply:
killed these